src/contexts/globalLayout/EditGalleryNavbar/OnboardingEditGalleryNavbar.tsx

```diff
@@ -37,6 +38,21 @@ export function OnboardingEditGalleryNavbar({
 }: Props) {
   const isMobile = useIsMobileOrMobileLargeWindowWidth();
 
+  useEffect(() => {
+    function handlePress(event: KeyboardEvent) {
+      if (event.key === 's' && event.metaKey) {
+        // Avoid trying to save page as HTML
+        event.preventDefault();
+
+        onDone();
+      }
+    }
+
+    window.addEventListener('keydown', handlePress);
+
+    return () => window.removeEventListener('keydown', handlePress);
+  }, [onDone]);
```

Comment:
can we use our existing `useMultiKeyDown`?

Reply:
i think we should toss that and rely on something more battle tested. https://react-hotkeys-hook.vercel.app/

Reply:
will do this in a separate pr right now